src/guides/configuration.md

```diff
@@ -0,0 +1,58 @@
+# Configuration
+
+Glimmer apps work right out of the box, but you can configure them to fit your needs.
+
+## Root Container
+
+Glimmer apps contain an `index.html` file that has the following div, which is where all your markup will reside in:
+
+```html
+<!-- ui/index.html -->
+<div id="app">
+  <!--Your app will render here at runtime-->
+</div>
+```
+
+If you'd like to change the container, open `ui/index.ts` and change
+
+```js
+const containerElement = document.getElementById('app');
+```
+
+to
+
+```js
+const containerElement = document.getElementById('conference-speaker-app');
+```
+
+Your host page can now hold multiple glimmer apps if you'd like. It can even point to other hosted scripts (that you trust) that other teams might host so you can get the latest updates without integration.
```

Comment:
Meaning if you leave only `div#app` and don't change the `containerElement`, they'll both render there? Not from my experience, one wins out and even if they both rendered, how would you decide which comes first in the DOM? I see your point about not being clear on how to link up with the 2nd component. I'll update.